py3status/modules/twitch.py

```diff
@@ -6,15 +6,52 @@
         (default 60)
     client_id: Your client id. Create your own key at https://dev.twitch.tv
         (default None)
+    client_secret: Your client secret.
+        (default None)
     format: Display format when online
         (default "{display_name} is live!")
     format_offline: Display format when offline
         (default "{display_name} is offline.")
+    format_tag: Tag formatting
+        (default "{name}")
+    locale: List of locales to try for tag translations, eg. ["cs-cz", "en-uk", "en-us"]. If none is specified, auto-detect from environment, with a fallback to "en-us".
```

Comment:
can we make this plural since this is a list please?

Reply:
done